sites/postcss-preset-env/rollup.config.mjs

```diff
@@ -11,6 +11,7 @@ const production = !process.env.ROLLUP_WATCH;
 export default [
 	'scripts',
 	'playground',
+	'blog_calc_2023_02_21',
```

Comment:
Using a very specific name in case we do more of these micro scripts for blogs posts.

I don't like the underscore but it gave an error when using a regular dash.

Reply:
Nicely thought no way to clash like this!